docs/user_guide/da_vinci_client/da_vinci_record_transformer.md

```diff
@@ -0,0 +1,41 @@
+---
+layout: default
+title: Da Vinci Record Transformer
+parent: Da Vinci Client
+grand_parent: User Guides
+permalink: /docs/user_guide/da_vinci_client/da_vinci_record_transformer
+---
+
+# Da Vinci Record Transformer
+
+This feature enables applications to transform records as they're being consumed and stored in the Da Vinci Client. It's capable of handling records that are compressed and/or chunked.
+
+## Usage
+To use the record transformer, you will need to implement the `DaVinciRecordTransformer` interface and pass it into the `StoreIngestionTask` constructor. When a message is being consumed, the `DaVinciRecordTransformer` will modify the value before it is written to storage.
```

Comment:
Nitpick: we want to respect a 120 characters per line limit in the docs as well. This makes it easier to comment on a specific part of the text, since GH only supports targeting at the granularity of entire lines when commenting. The markdown format ends up being rendered correctly with overflow and everything out of the box, no matter where we cut the lines.

For the `DaVinciRecordTransformer`, I would prefer that we put a link to the online JavaDoc, e.g. http://venicedb.org/javadoc/com/linkedin/davinci/client/DaVinciRecordTransformer.html in this case.

We should mention that the `DVRT` needs to be passed into the `DaVinciConfig`, not into the `SIT` (JavaDoc link: http://venicedb.org/javadoc/com/linkedin/davinci/client/DaVinciConfig.html#setRecordTransformer(com.linkedin.davinci.client.DaVinciRecordTransformer) ...). The transformer eventually makes its way down into the `SIT`, but that is an implementation detail, and we hope to keep this part of the documentation user-facing.